learningequality/studio
# Clipboard deletion

## Description

* Properly sets extra_fields on clipboard nodes when deleting child nodes from them
* Uses the same dynamic annotated field aggregation from content nodes to show counts on clipboard nodes

#### Issue Addressed (if applicable)

* Fixes #2472

## Steps to Test

- Copy a topic with content to the clipboard
- Delete a node from inside the topic in the clipboard
- Check that the clipboard still displays
- Check that the resource count is properly updated
